runtime/gc_structs/VMThreadStackSlotIterator.cpp

```diff
@@ -136,15 +136,13 @@ GC_VMThreadStackSlotIterator::scanSlots(
 			void *userData,
```

Comment:
This method is specific for Continuation object scan, and it should have really had more specific name  (scanSlotsForContinuation or scanContinuationSlots or something...). Since, you changed the signature and touched all calling sites, it's good moment to rename it now.